algorithm-prototyping/brute-force-prototype/src/main/java/edu/uga/devdogs/bruteforceprototype/schedule/ScheduleUtil.java

```diff
@@ -68,7 +68,39 @@ public static double computeAverageIdleTime(Schedule schedule) {
      * @return the overall objective score for the schedule
      */
     public static double computeOverallObjective(Schedule schedule, Map<String, Map<String, Double>> distances, double[] weights) {
-        return 0.0;
```

Comment:
Add checks to ensure that the parameters are valid. The parameters `schedule` and `distances` should not be null and `weights` should be an array of size 3. Any of these conditions not being met should result in an `IllegalArgumentException` specifying the problem.